web/modules/custom/rdf_entity/src/Entity/RdfEntitySparqlStorage.php

```diff
@@ -734,8 +735,9 @@ protected function doSave($id, EntityInterface $entity) {
       $id = $this->generateId();
       $entity->{$this->idKey} = (string) $id;
     }
-    elseif ($entity->isNew() && $this->idExists($id)) {
-      throw new \InvalidArgumentException("Attempting to create a new entity with the ID '$id' already taken.");
+    // The ID cannot overlap an existing 'rdf_entity' or 'taxonomy_term'.
```

Comment:
The storage layer can not/should not be aware of the entity types using it.
Also, issuing a full entity load seems overkill.
Even more important is that not all triples related to a mapped rdf:type.
(So the store might contain a lot more data then that is exposed in Drupal)